i18n/README.zh-cn.md

```diff
@@ -60,6 +61,16 @@ C++从头开始构建的，没有任何外部依赖，并且 100% 不受垃圾
 - 一个即時的加密货币（比特币、以太坊）交易数据集。
 - 一个包括 25 万艘船的时序地理数据集。
 
+| 查询                                                                          | 运行时间                                                                                                                                                                         |
+| ----------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
same as in English version. The queries are not using the real column names, but generic data type names. Then on the link they use the real query. In some of the queries we are also showing a different query (not just the column names) on the table and on the link. This is very confusing for me